Resources/Prototypes/_RMC14/Entities/Clothing/Belt/belts.yml

```diff
@@ -342,3 +395,218 @@
+#        ejectSound: /Audio/Weapons/Guns/MagOut/smg_magout.ogg
+        whitelist:
+          tags:
+          - WeaponSMGM63
+  - type: CMHolster
+  - type: CMItemSlots
+
+- type: entity
+  parent: CMBeltMarine
+  id: RMCBeltHolsterSMGPouch
+  name: M276 pattern M63 holster rig and pouch
+  description: Special issue variant of the M276 designed to holster a M63 submachine gun and two spare magazines. Uncommonly issued to USCM support and specialist personnel.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Clothing/Belt/holster_smg_pouch.rsi
+    layers:
+    - state: icon
+    - sprite: _RMC14/Objects/Weapons/Guns/gun_underlays.rsi
+      state: m63
+      offset: -0.34375, 0.0
+      map: [ "enum.CMItemSlotsLayers.Filled" ]
+      visible: false
+    - state: icon-front
+    - state: half
+      map: [ "openLayer" ]
+    - state: full
+      map: [ "closedLayer" ]
+  - type: Clothing
+    sprite: _RMC14/Objects/Clothing/Belt/holster_smg_pouch.rsi
+  - type: Storage # holds up to 2 mags plus gun
+    maxItemSize: Normal
+    grid:
+    - 0,0,3,1 # 2 slots
+    whitelist:
+      tags:
+      - CMMagazineSmg
+  - type: IgnoreContentsSize
+    items:
+      tags:
+      - CMMagazineSmg
+  - type: FixedItemSizeStorage
+  - type: ContainerContainer
+    containers:
+      pouch: !type:ContainerSlot
+  - type: ItemSlots
+    slots:
+      pouch:
+        name: M63 holster
+#        insertSound: /Audio/Weapons/Guns/MagIn/smg_magin.ogg # TODO RMC14 sounds
+#        ejectSound: /Audio/Weapons/Guns/MagOut/smg_magout.ogg
+        whitelist:
+          tags:
+          - WeaponSMGM63
+  - type: CMHolster
+  - type: CMItemSlots
+
+- type: entity
+  parent: CMBeltMarine
+  id: RMCBeltHolsterFlareGun
+  name: M276 pattern M82F flare gun holster rig
+  description: The M276 is the standard load-bearing equipment of the USCM. It consists of a modular belt with various clips. This version is for the M82F flare gun.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Clothing/Belt/holster_flaregun.rsi
+    layers:
+    - sprite: _RMC14/Objects/Weapons/Guns/gun_underlays.rsi
+      state: m82f
+      offset: -0.0625, 0.0
+      map: [ "enum.CMItemSlotsLayers.Filled" ]
+      visible: false
+    - state: icon
+    - state: half
+      map: [ "openLayer" ]
+    - state: full
+      map: [ "closedLayer" ]
+  - type: Clothing
+    sprite: _RMC14/Objects/Clothing/Belt/holster_flaregun.rsi
+  - type: Storage
+    grid:
+    - 0,0,19,3 # 20 flares
+    whitelist:
+      tags:
+      - Flare # todo: replace with flaregun ammo
+  - type: ContainerContainer
+    containers:
+      pouch: !type:ContainerSlot
+  - type: ItemSlots
+    slots:
+      pouch:
+        name: M82F holster
+#        insertSound: /Audio/Weapons/Guns/MagIn/smg_magin.ogg # TODO RMC14 sounds
+#        ejectSound: /Audio/Weapons/Guns/MagOut/smg_magout.ogg
+        whitelist:
+          tags:
+          - Sidearm # todo: replace with flaregun
```

Comment:
- RMCFlareGun

Reply:
I split the flare gun belt into a separate PR and added the RMCFlareGun tag to its holster's whitelist.